cachecontrol/cache.py

```diff
@@ -37,3 +37,7 @@ def delete(self, key):
         with self.lock:
             if key in self.data:
                 self.data.pop(key)
+
+    def clear(self):
```

Comment:
I understand that needed to use this method as a user of this backend? I'd go ahead and just add it to the cache back end and advertise its availability for users. I think you also said you fall back to the DictCache, in which case, you could also adjust the import error handling to return a compatible DictCache based cache if necessary.

Reply:
So you're saying you'd rather not have a `clear()` method in `DictCache`, and that I should just subclass it & add the method if needed ? That's ok with me, I'm just not 100% sure I understood your comment, English is not my 1st language.